test/performance/bench_common.h

```diff
@@ -108,10 +108,131 @@ void bench_suite_init(bench_suite_t *suite);
 void bench_run_indef(bench_info_t *info, odp_atomic_u32_t *exit_thread);
 
 /**
- * Run tests suite and print results
+ * Run test suite and print results
+ *
+ * The argument is of type 'bench_suite_t *'.
+ *
+ * @retval 0 on success
+ * @retval <0 on failure
  */
 int bench_run(void *arg);
```

Comment:
Doxygen tags (/**, @retval, ...) could be removed from comments as this file does not seem to be part of doxygen generated documentation (no doxygen errors from missing @param).

Reply:
Doxygen format removed in V2.